llvm/test/CodeGen/AArch64/sve-unpred-loads-stores.ll

```diff
@@ -0,0 +1,483 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -aarch64-sve-vector-bits-max=0   < %s | FileCheck %s --check-prefix=CHECK-VLA
```

Comment:
Given the general SVE code generation should already be well tested.  Does the `-aarch64-sve-vector-bits-max=0` RUN line offer any new value?

Reply:
Thanks, I was initially planning to use it to check the general SVE folds to LDR/STR, but you're right, there's already enough coverage for those. I've removed the `-aarch64-sve-vector-bits-max=0` RUN line and cleaned up the tests a bit.